extensions/src/ai_storybook_dancing/index.ts

```diff
@@ -43,6 +43,34 @@ export default class ExtensionNameGoesHere extends extension(details, "blockly",
     await dance("hop");
   }
 
+  @block({
+    text: "Step left",
+    type: "command"
+  })
+  async stepLeft() {
+    await dance("swivel left");
+  }
+
+  @block({
+    text: "Step right",
+    type: "command"
+  })
+  async stepRight() {
+    await dance("swivel right");
+  }
+
+  @block({
+    text: (direction: string) => `Spin ${direction}`,
+    type: "command",
+    arg: {
+      type: 'string',
+      options: ['left', 'right']
+    }
+  })
+  async spin(direction: string) {
+    await dance("swivel " + direction);
```

Comment:
Actually there's an explicit `spin` dance move (different than the swivels). See: https://github.com/mitmedialab/prg-ai-storybook-unity/blob/main/Assets/Scripts/Dancing/DanceMoves.cs#L111 (NOTE: that function will need to be updated to take an input direction when you work on [AS-5](https://mitjibo.atlassian.net/browse/AS-5))

So this should be updated to send `spin ${direction}` instead of swivel

Reply:
Oops! This one was a typo